string[]  group_names
---
meka_posture_recorder/PostureRecordErrorCodes error_code
